doc/guides/nest2_to_nest3/nest2_to_nest3_overview.rst

```diff
@@ -543,7 +543,7 @@ Setting and getting attributes directly
     as the SynapseCollection.
     
     For :ref:`spatially distributed <topo_changes>` sources and targets, you can access the distance between
-    the source, target pair by calling `distance` on your SynapseCollection
+    the source, target pair by calling `distance` on your SynapseCollection.
```

Comment:
```suggestion
    the source-target pairs by calling `distance` on your SynapseCollection.
```
Changing consistently from 'source, target pair' to 'source-target pairs' as pointed out by @jhnnsnk.

Reply:
Thanks @jasperalbers, didn't notice that one :+1: